### ISSUE TITLE
Отсутствует проект.

### ISSUE THREAD

**Issue by 8symbols** [quiksilver4210/jsjs#1]
**Описание:** при заходе в репозиторий он оказывается пустой.
**Ожидаемое поведение:** в репозитории должен присутствовать код.
**Когда повторяется ошибка:** всегда.
![image](https://user-images.githubusercontent.com/32729802/41515841-a2731da8-72f4-11e8-8274-46828b2a967b.png)

**Comment by Vixorem** [comment]
I agree with mr.8symbols. You've got no source code in this repository. Please, do something about it.
Best wishes, guthub user who cares about your profile.
